README.md

```diff
@@ -46,7 +46,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo](https://soccerhub.onrender.com/)
+- [Live Demo](https://www.loom.com/share/95292335049d41549cf61d8cb9a2b95a?sid=4af47703-6bd8-4ea2-935a-12b355878f7b)
 
```

Comment:
Nice job so far, your video presentation looks good👍🏼. In a bit to implement the corrections from the previous reviewer, I noticed you replaced the Live demo link with the video presentation. Kindly note that it is also a requirement that your readme file has a live demo link (A link to your deployed project) just as it should have a video presentation link. Kindly Add the link to your deployed project in your live demo link. Add a section called video presentation and add the link to your video presentation.